get_data.py

```diff
@@ -0,0 +1,108 @@
+"""Module to Store Data From the CSV files into data Structures and Handle the Parsed Commands"""
+
+import os
+import csv
+from daily_weather_data import WeatherDataRow
+
+
+class WeatherData:
+    """Class containing 2 class variables (lists) to use .
+    Contains attributes for file path , year to display and the month for each object
+    Contains a method to load monthly and yearly data
+    Contains an internal function which cuts out useless columns from the Monthly_weather dictionary and yearly_weather dictionaries
+    """
+
+    columns_to_keep = [
+        "PKT",
+        "Max TemperatureC",
+        "Mean TemperatureC",
+        "Min TemperatureC",
+        "Max Humidity",
+        " Mean Humidity",
+    ]
+    months_of_the_year = [
+        "jan",
+        "feb",
+        "mar",
+        "apr",
+        "may",
+        "jun",
+        "jul",
+        "aug",
+        "sep",
+        "oct",
+        "nov",
+        "dec",
+    ]
+
+    def __init__(self, parser_file_path, year, month):
+        """initialize function to store attributes affter instantiation of a class"""
+
+        self.parser_file_path = parser_file_path
+        self.year = year
+        self.month = month
+
+    def load_data(self, month=None):
```

Comment:
Again is this `month` parameter any different than `self.month`?

Reply:
Not really but i added it here to differentiate between yearly and monthly data . when a month is entered the value changes from none otherwise it will loop over all 12 months